R/draw-dm.R

```diff
@@ -32,6 +32,11 @@ cdm_draw <- function(dm,
                      focus = NULL,
                      graph_name = "Data Model") {
 
+  check_dm(dm)
+  if (is_empty(dm)) {
+    message("Cannot draw empty `dm`.")
```

Comment:
```suggestion
message("The dm cannot be drawn because it is empty.")
```

Reply:
Can you please keep indentation in your suggested change?

Reply:
Yes, certainly.